File: Python For Everyone, Horstmann/Chapter2/P2.7.py
# Write a program that prompts the user for a radius and then prints
# • The area and circumference of a circle with that radius
# • The volume and surface area of a sphere with that radius

radius = float(input("Enter the radius: "))

Pi = 3.14159265359

#circle
areaCircle = (radius * radius ) * Pi
diameterCircle = 2 * radius
circumferenceCircle = Pi * diameterCircle
#sphere
volumeSphere = (4 / 3) * Pi * (radius ** 3)
volumeArea = 4 * Pi * (radius * radius)


print("Circle area:          %.2f" % areaCircle)
print("Circle diameter:      %.2f" % diameterCircle)
print("Circle circumference: %.2f" % circumferenceCircle)
print("-" * 28)
print("Sphere volume:        %.2f" % volumeSphere)
print("Sphere area:          %.2f " % volumeArea)

File: Big-Java-for-Everyone-Late-Objects-Horstmann/Chapter-2-Fundamental-Data-Types/P2_7.java
/*Write a program that prompts the user for a radius and then prints
 • The area and circumference of a circle with that radius
 • The volume and surface area of a sphere with that radius*/


import java.util.Scanner;


public class P2_7 {
	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		System.out.print("Radius: ");
		double radius = input.nextDouble();
		input.close();
		System.out.println("Area of circle: " + (Math.PI * Math.pow(radius, 2)));
		System.out.println("Circumference of circle: " + (2 * Math.PI * radius));
		System.out.println("Volume of sphere: " + (4.0 / 3.0 * Math.PI * Math.pow(radius, 3)));
		System.out.println("Surface area of sphere: "+ (4.0 * Math.PI * Math.pow(radius, 2)));
	}
}
